resources/views/profile/delete-user-form.blade.php

```diff
@@ -23,41 +23,53 @@
             <x-slot name="description">
                 <div class="flex flex-col">
                     <div class="flex justify-center w-full">
-                        <x-ark-icon name="fortify-modal.delete-account" class="mt-8 mb-4 w-60 h-auto text-theme-primary-600 light-dark-icon"/>
+                        <x-ark-icon :name="$icon ?? 'fortify-modal.delete-account'" class="mt-8 mb-4 w-60 h-auto text-theme-primary-600 light-dark-icon"/>
```

Comment:
in what way does the icon differ for the deployer? it should all be using the same image for this modal really as it's supposed to be generic